modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -303,6 +299,23 @@ private void _excludeMethods(
 				}
 			}
 
+			Set<String> currentExcludedOperationIds =
+				ConfigurationUtil.getExcludedOperationIds(
+					objectDefinition.getCompanyId(), _configurationAdmin,
+					objectDefinition.getRESTContextPath());
+
+			if (Objects.equals(
+					currentExcludedOperationIds, excludedOperationIds)) {
+
+				return;
+			}
+
+			Configuration configuration =
```

Comment:
I know it is just a stupid idea, but I think we could sort the excludedOperationIds alphabetically when we are setting the new configuration; I think it will be easier to find a operationId inside in that case. What do you think @guilhermedcamacho ?

Reply:
I've kept the current behavior, but if you find it necessary to make this improvement,, I can do that sorting. Should I proceed?